server.php

```diff
@@ -65,9 +65,11 @@ function valet_support_xip_io($domain)
 $valetSitePath = null;
 
 foreach ($valetConfig['paths'] as $path) {
-    if (is_dir($path.'/'.$siteName)) {
-        $valetSitePath = $path.'/'.$siteName;
-
+    $domain = ($pos = strrpos($siteName, '.')) !== false
+                ? substr($siteName, $pos+1)
+                : null;
+    if (is_dir($path.'/'.$siteName) || is_dir($path.'/'.$domain)) {
```

Comment:
I believe this needs to be changed to `if (is_dir($path.'/'.$siteName) || (is_dir($path.'/'.$domain) && $domain))`. Without this, it just resolves `~/.valet/Sites`, which is always a directory. Meaning sub domains work, but top level domains do not. This fixes that.